src/BKTExceptions.ts

```diff
@@ -4,16 +4,17 @@ import {
 } from './internal/model/MetricsEventData'
 
 abstract class BKTBaseException extends Error {
+  name = 'BKTBaseException'
```

Comment:
BKTBaseException name property is not given `as const` because it cannot be overridden by subclasses if `as const` is given.